pkg/operation/care/checker.go

```diff
@@ -53,8 +53,7 @@ var (
 	)
 
 	requiredMonitoringSeedDeployments = sets.NewString(
-		v1beta1constants.DeploymentNameGrafanaOperators,
-		v1beta1constants.DeploymentNameGrafanaUsers,
+		v1beta1constants.DeploymentNameGrafana,
```

Comment:
Hi @ialidzhikov, that is a good catch, thank you! We did not think about it because in the local setup the gardenlet update triggers a shoot reconciliation probably because the shoot does not have a maintenance time window. We'll have to check how to improve this. cc @rickardsjp

Reply:
Hi @ialidzhikov, please see the commit https://github.com/gardener/gardener/pull/7007/commits/58f4aa87f8de73fa06ee2efa956b209a88e18fec "Improve the ObservabilityComponentsHealthy condition for the transition phase".
I could reproduce the issue that you pointed out above in my local setup and could confirm that with the new commit the health check properly considers the version of the shoot. Thank you for spotting this mistake!

Reply:
Thanks for addressing!